generate-votes-list/voting.js

```diff
@@ -94,11 +115,11 @@ async function setupProgressBar() {
 }
 
 function checkConfig() {
-    if (fs.existsSync("./.cfg") && fs.existsSync("./votes.json")) {
+    if (fs.existsSync("./.cfg") && fs.existsSync("./ethVotes.json")) {
```

Comment:
the ae file is `ae-votes`, so maybe `eth-votes`

Reply:
good idea!